InternTrack.Core.Api/Models/Interns/Intern.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+
+using System;
+
+namespace InternTrack.Core.Api.Models.Interns
+{
+    public class Intern
+    {
+        public Guid Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public string PhoneNumber { get; set; }
+        public string Status { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
+        public DateTimeOffset JoinDate { get; set; }
+        public Guid CreatedBy { get; set; }
+        public Guid UpdatedBy { get; set; }
+        public String TimeZone { get; set; }
```

Comment:
Why do we need that?

Reply:
I talked to @Manasa1191 and she agreed and she will redo the migration
@ADILATIC make sure you start your PRs after @Manasa1191  is done.